conversions/decimal_to_hexadecimal.py

```diff
@@ -21,23 +21,60 @@
 }
 
 def decimal_to_hexadecimal(decimal):
-    """ take decimal value, return hexadecimal representation as str """
+    """
+        take integer decimal value, return hexadecimal representation as str 
+        >>> decimal_to_hexadecimal(5)
+        '5'
```

Comment:
I believe that you can drop the quotes if you want.

Reply:
When I drop the quotes, the doctest errors.  This is my first time using doctest so I do not know if there is a workaround.